tests/pull_requests.json

```diff
@@ -0,0 +1,370 @@
+            "labels_url": "https://api.github.com/repos/simonw/datasette/labels{/name}",
+            "releases_url": "https://api.github.com/repos/simonw/datasette/releases{/id}",
+            "deployments_url": "https://api.github.com/repos/simonw/datasette/deployments",
+            "created_at": "2017-10-23T00:39:03Z",
+            "updated_at": "2020-07-27T20:42:15Z",
+            "pushed_at": "2020-07-26T01:21:05Z",
+            "git_url": "git://github.com/simonw/datasette.git",
+            "ssh_url": "git@github.com:simonw/datasette.git",
+            "clone_url": "https://github.com/simonw/datasette.git",
+            "svn_url": "https://github.com/simonw/datasette",
+            "homepage": "http://datasette.readthedocs.io/",
+            "size": 3487,
+            "stargazers_count": 3642,
+            "watchers_count": 3642,
+            "language": "Python",
+            "has_issues": true,
+            "has_projects": false,
+            "has_downloads": true,
+            "has_wiki": true,
+            "has_pages": false,
+            "forks_count": 206,
+            "mirror_url": null,
+            "archived": false,
+            "disabled": false,
+            "open_issues_count": 190,
+            "license": {
+              "key": "apache-2.0",
+              "name": "Apache License 2.0",
+              "spdx_id": "Apache-2.0",
+              "url": "https://api.github.com/licenses/apache-2.0",
+              "node_id": "MDc6TGljZW5zZTI="
+            },
+            "forks": 206,
+            "open_issues": 190,
+            "watchers": 3642,
+            "default_branch": "master"
+          }
+        },
+        "_links": {
+          "self": {
+            "href": "https://api.github.com/repos/simonw/datasette/pulls/571"
+          },
+          "html": {
+            "href": "https://github.com/simonw/datasette/pull/571"
+          },
+          "issue": {
+            "href": "https://api.github.com/repos/simonw/datasette/issues/571"
+          },
+          "comments": {
+            "href": "https://api.github.com/repos/simonw/datasette/issues/571/comments"
+          },
+          "review_comments": {
+            "href": "https://api.github.com/repos/simonw/datasette/pulls/571/comments"
+          },
+          "review_comment": {
+            "href": "https://api.github.com/repos/simonw/datasette/pulls/comments{/number}"
+          },
+          "commits": {
+            "href": "https://api.github.com/repos/simonw/datasette/pulls/571/commits"
+          },
+          "statuses": {
+            "href": "https://api.github.com/repos/simonw/datasette/statuses/a85239f69261c10f1a9f90514c8b5d113cb94585"
+          }
+        },
+        "author_association": "OWNER",
+        "active_lock_reason": null,
+        "merged": true,
+        "mergeable": null,
+        "rebaseable": null,
+        "mergeable_state": "unknown",
+        "merged_by": {
```

Comment:
Running this should create a `merged_by` column on the `pull_requests` table which is a foreign key to the `users` table.

Reply:
Nice eye. This is a little misleading. The merged by field is only available for specified PRs (singular), which is how I generated this. There are a different set of keys when requesting PRs (plural) via the API. Happy to modify the payload here, but merged_by is not included. For reference, [here is a gist](https://gist.github.com/adamjonas/5ee0ce97f091e8e36931d5e8d7eb6502) of what the first PR from the pull-requests request looks like.